src/adapters/auth-adapters.ts

```diff
@@ -0,0 +1,118 @@
+import {getCookie , setCookie, deleteCookie} from '../util/Authentication'
+
+const API_URL = 'http://localhost:8000/api/v1'
+let endpoint = ''
+export const fetchAPIData = async (url: string, method: string, body:object , headers?:object) => {
+    const token = getCookie('shelf-share-session')
```

Comment:
Our cookie should actually be an object. Token will be a key on the object. Instead of const token here I would say const cookie. Then on a new line you can say const token = cookie.token.